traffic_ops/app/lib/UI/SslKeys.pm

```diff
@@ -34,7 +34,7 @@ sub add {
 	&stash_role($self);
 
 	#get key data from keystore
-	my $response_container = $self->riak_get( 'ssl', "$xml_id-latest");
+	my $response_container = $self->riak_get( 'ssl', "ds_$ds_id-latest");
```

Comment:
a preexisting key won't be found and won't get stashed. I'm not sure how much of an issue that might be.